src/main/kotlin/listener/CommandListener.kt

```
@@ -48,6 +49,8 @@ class CommandListener @ListenerConstructor constructor(private val jda: JDA) : L
                 }
             } catch (e: CommandExitException) {
                 event.channel.sendMessage(e.message!!).queue()
+            } catch (e: DatabaseException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)